app/models/version.rb

```diff
@@ -139,7 +139,7 @@ def self.new_pushed_versions(limit = 5)
   end
 
   def self.just_updated(limit = 5)
-    subquery = <<-SQL
+    subquery = <<~SQL.squish
```

Comment:
Is the `squish` still needed? The whole idea of replacing `<<-` with `<<~` was to avoid squish at all.

Reply:
yeah, it is complaining:
```
app/models/version.rb:132:16: C: Rails/SquishedSQLHeredocs: Use <<~SQL.squish instead of <<~SQL. (https://rails.rubystyle.guide/#squished-heredocs)
    subquery = <<~SQL
               ^^^^^^
````
I think the squiggly operator only removes the indentation. squish also move everything to a single line.

Reply:
ahh, ok, sorry for confusion